src/app/helptext/network/interfaces/interfaces-form.ts

```diff
@@ -60,4 +56,65 @@ alias_netmaskbit6_tooltip : T('Select the prefix length used on the network.'),
 delete_placeholder6: T('Delete'),
 delete_tooltip6: T('Set to delete this alias.'),
 
+bridge_members_placeholder: T('Bridge Members'),
+bridge_members_tooltip: T(''),
+
+failover_critical_placeholder: T('Failover Critical'),
+failover_critical_tooltip: T(''),
+
+failover_group_placeholder: T('Failover Group'),
+failover_group_tooltip: T(''),
+
+failover_vhid_placeholder: T('Failover VHID'),
+failover_vhid_tooltip: T(''),
+
+failover_alias_address_placeholder: T('Failover Alias'),
+failover_alias_address_tooltip: T(''),
```

Comment:
From TrueNAS Guide "Failover" section:

"IPv4 address to use for the failover interface. Set a manual address here if DHCP will not be used."

Reply:
Should probably call IPs.

These are the IPs of the controller 2/B.

Reply:
These fields are able to take either IPv4 or IPv6 with the new page

Reply:
The backend does not really support IPv6 for failover yet ;) It may accept but does not work

Reply:
Ah :)

Reply:
Same for this one as below, "IP Address" instead of "Alias"? @wblock  @william-gr

Reply:
yes